src/coreclr/src/md/ceefilegen/ceesectionstring.cpp

```diff
@@ -67,8 +67,6 @@ CeeSectionString::~CeeSectionString()
 StringTableEntry* CeeSectionString::createEntry(__in_z LPWSTR target, ULONG hashId)
 {
     StringTableEntry *entry = new StringTableEntry;
```

Comment:
Actually, I think a better fix would be to use `new (nothrow)` here. The calling code does not seem to expect exceptions to be thrown from here.